tests/Resources/ti.ui.tableview.test.js

```diff
@@ -1207,68 +1213,77 @@ describe('Titanium.UI.TableView', function () {
 		refreshControl.addEventListener('refreshend', () => {
 			finish();
 		});
-		win = Ti.UI.createWindow();
-		win.add(Ti.UI.createListView({
-			refreshControl: refreshControl
-		}));
+    
 		win.addEventListener('open', function () {
 			refreshControl.beginRefreshing();
 		});
+    
+    win.add(Ti.UI.createListView({ refreshControl: refreshControl }));
```

Comment:
<!--
  1 failure:  tests/Resources/t...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.ui.tableview.test.js;
  Line: 1221;
-->
- :no_entry_sign: tests/Resources/ti.ui.tableview.test.js line 1221 – Expected indentation of 2 tabs but found 4 spaces. (indent)